### PR TITLE
Update eo.js

### PR DESCRIPTION
**Pull request by Vivakvo**
Aldono de la litero "t" por marto kaj septembro por distingi ilin de mardo kaj sep=7 (linio 13).
Korekto de kelkaj esprimoj, en kiuj mankis akuzativo (linioj 23, 39 kaj 41).
Aldono de "kelkaj" por "s" (linio 47). Aldono de "unu" por "m", "h", "d", "M" kaj "y" (linioj 49 51 53 55 kaj 57), ĉar temas pri la kvanto "1", male al nedifinita afero.

### CODE REVIEW

**Comment by ichernev** [comment]
@Vivakvo you need to change files under `src/locale/eo.js` and `src/test/locale/eo.js`, not `locale/*.js`.
@Vivakvo you also need to provide some proof that your changes are correct (official sources).
Also you can try to speak English.

@colindean @miestasmia what do you think?

**Comment by colindean** [comment]
Mi aprobas kun la suggesto de @ichernev kompletiĝonta! 

I approve with @ichernev's suggestion about to be completed!

**Comment by Vivakvo** [comment]
Thank you for your comments. I'm pleased to see some humans here.  
Thank you for your help, I don't know this github very well and I am going to put the file at the right place.  
About proof, first I can say that I'm speaking Esperanto for 30 years, and I teach it for about 20 years. I can try to explain the reasons of the changes I propose, but are there some Esperanto speakers here to understand and approve them ?

File: locale/eo.js [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
     var eo = moment.defineLocale('eo', {
         months : 'januaro_februaro_marto_aprilo_majo_junio_julio_aŭgusto_septembro_oktobro_novembro_decembro'.split('_'),
-        monthsShort : 'jan_feb_mar_apr_maj_jun_jul_aŭg_sep_okt_nov_dec'.split('_'),
+        monthsShort : 'jan_feb_mart_apr_maj_jun_jul_aŭg_sept_okt_nov_dec'.split('_'),
```
Comment (Vivakvo) [inline locale/eo.js:13]: Adding the letter "t" for the months "marto" and "septembro" to distinguish them from "mardo" (=tuesday) and "sep" (=seven)

File: locale/eo.js [hunk 2]
```diff
@@ -20,7 +20,7 @@
             L : 'YYYY-MM-DD',
             LL : 'D[-a de] MMMM, YYYY',
             LLL : 'D[-a de] MMMM, YYYY HH:mm',
-            LLLL : 'dddd, [la] D[-a de] MMMM, YYYY HH:mm'
+            LLLL : 'dddd[n], [la] D[-an de] MMMM, YYYY HH:mm'
```
Comment (Vivakvo) [inline locale/eo.js:23]: Correction of some expressions in which accusative is needed.

File: locale/eo.js [hunk 3]
```diff
@@ -36,25 +36,25 @@
         calendar : {
             sameDay : '[Hodiaŭ je] LT',
             nextDay : '[Morgaŭ je] LT',
-            nextWeek : 'dddd [je] LT',
+            nextWeek : 'dddd[n je] LT',
             lastDay : '[Hieraŭ je] LT',
-            lastWeek : '[pasinta] dddd [je] LT',
+            lastWeek : '[pasintan] dddd[n je] LT',
```
Comment (Vivakvo) [inline locale/eo.js:41]: Correction of some expressions in which accusative is needed.

File: locale/eo.js [hunk 4]
```diff
@@ -36,25 +36,25 @@
...
             sameElse : 'L'
         },
         relativeTime : {
             future : 'post %s',
             past : 'antaŭ %s',
-            s : 'sekundoj',
+            s : 'kelkaj sekundoj',
```
Comment (Vivakvo) [inline locale/eo.js:47]: Adding the word "kelkaj" (=some), because "post sekundoj" and "antaŭ sekundoj" don't really make sense.

File: locale/eo.js [hunk 5]
```diff
@@ -36,25 +36,25 @@
...
+            h : 'unu horo',
             hh : '%d horoj',
-            d : 'tago',//ne 'diurno', ĉar estas uzita por proksimumo
+            d : 'unu tago',//ne 'diurno', ĉar estas uzita por proksimumo
             dd : '%d tagoj',
-            M : 'monato',
+            M : 'unu monato',
             MM : '%d monatoj',
-            y : 'jaro',
+            y : 'unu jaro',
```
Comment (Vivakvo) [inline locale/eo.js:57]: Adding the word "unu" (=one) because it's about a quantity, and not about an indefinite thing introduced by the article a/an.

**Comment by Vivakvo** [comment]
I had a problem to create a pull request for the file in src/locale/eo.js because I don't master the github.  
I have done some corrections, I have tried to commit but the pull request has gone I don't know where.
Could you help me?

**Comment by colindean** [comment]
I yield to @Vivakvo's experience. All of the suggestions are sensible.

@Vivakvo, can you make the appropriate changes two these two files?

1. https://github.com/Vivakvo/moment/blob/patch-1/src/locale/eo.js
2. https://github.com/Vivakvo/moment/blob/patch-1/src/test/locale/eo.js

I've already linked directly to the branch on your personal fork of `moment`, so you, in theory, need only hit _edit_ to make the changes.

Dankon!

**Comment by Vivakvo** [comment]
Hello @colindean !  
I have made the changes by the pull requests [5455](https://github.com/moment/moment/pull/5455) and [5456](https://github.com/moment/moment/pull/5456).
Is it suitable? Ĉu tio taŭgas kaj sufiĉas? Or must I make the changes again using the url you gave?
Amike,
Yves

**Comment by colindean** [comment]
Combined, they are correct but they must be submitted together in order to pass tests! Don't worry,  I'll handle that. However, there are some edge cases that your modifications don't handle:

```
Errors:

Module: locale:eo Test: format
llll ---> dim, la 14-an de feb, 2010 15:25
    at Object.<anonymous> (/Users/colin/Source/moment/build/umd/test/locale/eo.js:334:20)
Actual value:
dimn, la 14-an de feb, 2010 15:25
Expected value:
dim, la 14-an de feb, 2010 15:25

Module: locale:eo Test: fromNow
post kelkaj sekundoj
    at Object.<anonymous> (/Users/colin/Source/moment/build/umd/test/locale/eo.js:431:16)
Actual value:
post kelkaj sekundoj
Expected value:
post kelakj sekundoj

Module: locale:eo Test: calendar next week
Today + 2 days current time
    at Object.<anonymous> (/Users/colin/Source/moment/build/umd/test/locale/eo.js:451:20)
Actual value:
sabaton je 21:19
Expected value:
sabato je 21:19

Module: locale:eo Test: calendar next week
Today + 2 days beginning of day
    at Object.<anonymous> (/Users/colin/Source/moment/build/umd/test/locale/eo.js:453:20)
Actual value:
sabaton je 00:00
Expected value:
sabato je 00:00

Module: locale:eo Test: calendar next week
Today + 2 days end of day
    at Object.<anonymous> (/Users/colin/Source/moment/build/umd/test/locale/eo.js:455:20)
Actual value:
sabaton je 23:59
Expected value:
sabato je 23:59

Module: locale:eo Test: calendar next week
Today + 3 days current time
    at Object.<anonymous> (/Users/colin/Source/moment/build/umd/test/locale/eo.js:451:20)
Actual value:
dimanĉon je 21:19
Expected value:
dimanĉo je 21:19

Module: locale:eo Test: calendar next week
Today + 3 days beginning of day
    at Object.<anonymous> (/Users/colin/Source/moment/build/umd/test/locale/eo.js:453:20)
Actual value:
dimanĉon je 00:00
Expected value:
dimanĉo je 00:00

Module: locale:eo Test: calendar next week
Today + 3 days end of day
    at Object.<anonymous> (/Users/colin/Source/moment/build/umd/test/locale/eo.js:455:20)
Actual value:
dimanĉon je 23:59
Expected value:
dimanĉo je 23:59

Module: locale:eo Test: calendar next week
Today + 4 days current time
    at Object.<anonymous> (/Users/colin/Source/moment/build/umd/test/locale/eo.js:451:20)
Actual value:
lundon je 21:19
Expected value:
lundo je 21:19

Module: locale:eo Test: calendar next week
Today + 4 days beginning of day
    at Object.<anonymous> (/Users/colin/Source/moment/build/umd/test/locale/eo.js:453:20)
Actual value:
lundon je 00:00
Expected value:
lundo je 00:00

Module: locale:eo Test: calendar next week
Today + 4 days end of day
    at Object.<anonymous> (/Users/colin/Source/moment/build/umd/test/locale/eo.js:455:20)
Actual value:
lundon je 23:59
Expected value:
lundo je 23:59

Module: locale:eo Test: calendar next week
Today + 5 days current time
    at Object.<anonymous> (/Users/colin/Source/moment/build/umd/test/locale/eo.js:451:20)
Actual value:
mardon je 21:19
Expected value:
mardo je 21:19

Module: locale:eo Test: calendar next week
Today + 5 days beginning of day
    at Object.<anonymous> (/Users/colin/Source/moment/build/umd/test/locale/eo.js:453:20)
Actual value:
mardon je 00:00
Expected value:
mardo je 00:00

Module: locale:eo Test: calendar next week
Today + 5 days end of day
    at Object.<anonymous> (/Users/colin/Source/moment/build/umd/test/locale/eo.js:455:20)
Actual value:
mardon je 23:59
Expected value:
mardo je 23:59

Module: locale:eo Test: calendar next week
Today + 6 days current time
    at Object.<anonymous> (/Users/colin/Source/moment/build/umd/test/locale/eo.js:451:20)
Actual value:
merkredon je 21:19
Expected value:
merkredo je 21:19

Module: locale:eo Test: calendar next week
Today + 6 days beginning of day
    at Object.<anonymous> (/Users/colin/Source/moment/build/umd/test/locale/eo.js:453:20)
Actual value:
merkredon je 00:00
Expected value:
merkredo je 00:00

Module: locale:eo Test: calendar next week
Today + 6 days end of day
    at Object.<anonymous> (/Users/colin/Source/moment/build/umd/test/locale/eo.js:455:20)
Actual value:
merkredon je 23:59
Expected value:
merkredo je 23:59
```

I'm going to submit a clean PR with your #5455 and #5456 on the same branch, plus some commits that fix the problems I see.

**Comment by colindean** [comment]
@Vivakvo Which is correct, 1 or 2? Kiuj estas korekta?

1, your changes / viaj ŝanĝoj:

    dimn, la 14-an de feb, 2010 15:25

2: original / originalo:

    dim, la 14-an de feb, 2010 15:25

Mi ne memoras vidantan la tempmarkon kiel 1.

(Pardonu min, mia Esperanto estas sen ĵusa kutimo!)

**Comment by Vivakvo** [comment]
@colindean Dankon pro viaj rimarkoj kaj helpoj. Thank you for your help and comments.
Mi regas Esperanton sed ne tute programadon.
Between 1 and 2, of course only the original 2 is correct (no accusative to the short days). The accusative is put only to the long days (dimanĉon, la 14-an de feb...).
Could you correct this, because I don't know exactly where I should modify this (I saw LLLL but not llll in the file)?

**Comment by Vivakvo** [comment]
@colindean 
I didn't reply all the incorrect tests you have mentioned yesterday, but I suppose you didn't because you knew the right response. 
In particular, one must write "post kelkaj sekundoj" and not "post kelakj sekundoj" (tio estas tajperaro).
For the other dates, the right writing is with the accusative: dimanĉon, lundon, mardon, ktp.
